src/components/Icons/IconTypes.tsx

```diff
@@ -15,6 +15,9 @@ export enum IconNames {
     close = "close",
     download = "download",
     headset = "headset",
+    logo_brooklyn = "logo_brooklyn",
```

Comment:
I think instead of putting this in IconTypes let's make a new Type object called Logo.  Most of these icons are interchangeable, like "Hey you could put a search button magnifying glass here, or you could put a plus sign", but I can't think of any situation where you'd put the NYPL logo where you put a "close" button and vice versa.  So let's make a new type where people can choose between the three logos and nothing else.